admin/worker/worker.go

```diff
@@ -33,6 +33,9 @@ func New(logger *zap.Logger, adm *admin.Service) *Worker {
 func (w *Worker) Run(ctx context.Context) error {
 	group, ctx := errgroup.WithContext(ctx)
 	group.Go(func() error { return w.schedule(ctx, "check_slots", w.checkSlots, 15*time.Minute) })
+	group.Go(func() error {
+		return w.schedule(ctx, "delete_expired_device_auth_codes", w.deleteExpiredDeviceAuthCodes, 15*time.Minute)
```

Comment:
Not sure if we should run it this frequently as in the normal flow device code will be cleaned up after confirmation code approval or rejection. Only in cases where flow is abandoned in the middle code will be not be deleted. Probably running once in few hours should be sufficient ?